### PR TITLE
conf: verify_start_hooks() after lxc.mount.entry

### PR DESCRIPTION
**Pull request by brauner**
Fixes: https://discuss.linuxcontainers.org/t/are-lxc-mount-entry-available-when-lxc-hook-start-is-validated/2906/3
Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>

### CODE REVIEW

**Comment by joaocfernandes** [comment]
Thank you, for your fast reply.

File: src/lxc/conf.c [hunk 1]
```diff
@@ -3521,9 +3520,9 @@ static bool verify_start_hooks(struct lxc_conf *conf)
 		if (ret < 0 || ret >= PATH_MAX)
 			return false;
 
-		ret = stat(path, &st);
+		ret = access(path, F_OK);
```
Comment (stgraber) [inline src/lxc/conf.c]: Wouldn't X_OK actually be better since we'll be execing it?

Comment (brauner) [reply src/lxc/conf.c]: Hm, good point.

Comment (brauner) [reply src/lxc/conf.c]: updated

**Pull request merged**